docs/src/supported_releases.md

```diff
@@ -82,7 +82,7 @@ Git tags for versions are prefixed with `v`.
 <!-- TODO: Complete dates and versions below -->
 | Version         | Currently supported  | Release date        | End of life         | Supported Kubernetes versions | Tested, but not supported | Supported Postgres versions |
 |-----------------|----------------------|---------------------|---------------------|-------------------------------|---------------------------|-----------------------------|
-| 1.25.x          | Yes                  | December XX, 2024   | ~ February, 2025    | 1.29, 1.30, 1.31, 1.32 (??)   | 1.27, 1.28                | 13 - 17                     |
+| 1.25.x          | No (RC)              | December XX, 2024   | ~ February, 2025    | 1.29, 1.30, 1.31, 1.32 (!)    | 1.27, 1.28                | 13 - 17                     |
```

Comment:
The End of life here is wrong.
Below in the upcoming releases we say "May/Jun 2025".
By consistency, since in this table we use full month names.
```suggestion
| 1.25.x          | No (RC)              | December XX, 2024   | ~ May/June, 2025    | 1.29, 1.30, 1.31, 1.32 (!)    | 1.27, 1.28                | 13 - 17                     |
```